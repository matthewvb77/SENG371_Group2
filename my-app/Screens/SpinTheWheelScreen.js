import React from "react";
import { View, Text, StyleSheet, Image } from "react-native";

const SpinTheWheelScreen = () => {
	return (
		<View>
			<Text>Spin the Wheel</Text>
		</View>
	);
};

export default SpinTheWheelScreen;
