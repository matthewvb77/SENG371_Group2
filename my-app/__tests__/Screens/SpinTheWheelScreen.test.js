import React from "react";
import renderer from "react-test-renderer";

import SpinTheWheelScreen from "../../Screens/SpinTheWheelScreen";

describe("<SpinTheWheelScreen />", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<SpinTheWheelScreen />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
