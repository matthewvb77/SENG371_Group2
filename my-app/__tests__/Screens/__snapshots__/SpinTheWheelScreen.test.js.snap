// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpinTheWheelScreen /> renders correctly 1`] = `
<View>
  <Text>
    Spin the Wheel
  </Text>
</View>
`;
